test/core/end2end/h2_ssl_session_reuse_test.cc

```diff
@@ -263,9 +243,93 @@ TEST(H2SessionReuseTest, SingleReuse) {
 
   grpc_server* server = server_create(cq, server_addr.c_str());
 
-  do_round_trip(cq, server, server_addr.c_str(), cache, false);
-  do_round_trip(cq, server, server_addr.c_str(), cache, true);
-  do_round_trip(cq, server, server_addr.c_str(), cache, true);
+  // create client creds
+  grpc_slice ca_slice, cert_slice, key_slice;
+  GPR_ASSERT(GRPC_LOG_IF_ERROR("load_file",
+                               grpc_load_file(CA_CERT_PATH, 1, &ca_slice)));
+  GPR_ASSERT(GRPC_LOG_IF_ERROR(
+      "load_file", grpc_load_file(CLIENT_CERT_PATH, 1, &cert_slice)));
+  GPR_ASSERT(GRPC_LOG_IF_ERROR("load_file",
+                               grpc_load_file(CLIENT_KEY_PATH, 1, &key_slice)));
+  const char* ca_cert =
+      reinterpret_cast<const char*> GRPC_SLICE_START_PTR(ca_slice);
+  const char* client_cert =
+      reinterpret_cast<const char*> GRPC_SLICE_START_PTR(cert_slice);
+  const char* client_key =
+      reinterpret_cast<const char*> GRPC_SLICE_START_PTR(key_slice);
+  grpc_ssl_pem_key_cert_pair signed_client_key_cert_pair = {client_key,
+                                                            client_cert};
+  grpc_channel_credentials* client_creds = grpc_ssl_credentials_create(
```

Comment:
Is it possible to move the creation of the `grpc_channel_credentials` to a helper function so we don't have to duplicate the code?